index.bs

```diff
@@ -5793,7 +5796,7 @@ partial interface MLGraphBuilder {
   <summary>
     The <dfn method for=MLGraphBuilder>where(|condition|, |input|, |other|)</dfn> method steps are:
   </summary>
-    1. If |condition|'s [=MLOperand/dataType=] is not equal to {{MLOperandDataType/"uint8"}}, then [=exception/throw=] a {{TypeError}}.
+    1. If |condition|'s [=MLOperand/dataType=] is not {{MLOperandDataType/"uint8"}}, then [=exception/throw=] a {{TypeError}}.
```

Comment:
Hmm, if we're eliding `equal to` here, then do we need it above too?

```diff
- If its [=MLOperand/dataType=] is not equal to |input|'s [=MLOperand/dataType=], then [=exception/throw=] a {{TypeError}}.
+ If its [=MLOperand/dataType=] is not |input|'s [=MLOperand/dataType=], then [=exception/throw=] a {{TypeError}}.
```

Meh, shrug.

Reply:
Good test. Compare:

* _"If Bob's height is not 1.6m, then..."_  - seems fine
* _"If Bob's height is not equal to 1.6m, then..."_ - seems fine, but wordier
* _"If Bob's height is not equal to Alice's height, then..."_ - seems fine
* _"If Bob's height is not the same as Alice's height, then..."_ - also seems fine (but was used less frequently)
* _"If Bob's height is not Alice's height, then..."_ - reads a bit awkward to me, especially if it's not clear that this is a simple scalar value (e.g. is this literally a shared reference?)

Ultimately I'm fine with anything but aiming for consistency.

Reply:
These are fine in English, what is important is which is closer to spec'ese, or otherwise ecmascript'ese.
IIRC "is equal" and "is not equal" are used in most standards (e.g. HTML, INFRA, ...), to be checked.

Reply:
It gets more complicated when (1) the test is inverted and (2) there are multiple options - see other comment. But agreed that explicit "equal" is probably clearer.

Reply:
reverted in ef4eebc618228c4c281952a72aa15a2639f049bf